core/domain/collection_domain_test.py

```diff
@@ -1077,3 +1079,18 @@ def test_validation_fails_with_invalid_contributor_id_in_contributor_ids(
             utils.ValidationError,
             'Expected each id in contributor_ids to be string, received 2'):
             self.collection_summary.validate()
+
+    def test_is_private(self):
+        self.assertTrue(self.collection_summary.is_private())
+        self.collection_summary.status = constants.ACTIVITY_STATUS_PUBLIC
+        self.assertFalse(self.collection_summary.is_private())
+
+    def test_is_solely_owned_by_user(self):
```

Comment:
Not enough tests. Consider cases where the owner is not associated with the collection at all, the owner is an editor, etc. Pretend you are testing someone else's code and you have no insight into the implementation.